src/components/VContentSwitcher/VSearchTypes.vue

```diff
@@ -79,12 +78,9 @@ export default defineComponent({
       },
     ]
 
-    /**
-     * @todo This is for testing purposes only! We may want a different abstraction here;
-     * For example having all content types under `content.types` and making them filterable,
-     * like `const additional = [...content.types.filter(i => i.status === ADDITIONAL)]`.
-     */
-    if (isDev) {
+    const featureFlagStore = useFeatureFlagStore()
+
+    if (featureFlagStore.isOn('external_sources')) {
       content.additionalTypes = ['model_3d']
```

Comment:
`content` here refers to the return from the composable:

```
const searchTypes = [...supportedSearchTypes]


export default function useSearchType() {
  const activeType = computed(() => useSearchStore().searchType)
  const previousSearchType = ref(activeType.value)
  const setActiveType = (searchType) => {
    if (previousSearchType.value === searchType) return
    useSearchStore().setSearchType(searchType)
    previousSearchType.value = searchType
  }
  return {
    setActiveType,
    activeType,
    types: searchTypes,
    icons,
  }
}
```
And I can't see how we can add an `additionalTypes` property to a composable return object. Will it be available to other places where it's used?

Reply:
That's a good idea!
It's this part that we need to change:
`const searchTypes = [...supportedSearchTypes]`
It's currently set outside of the composable to keep the global state.

Reply:
We'll need to update all the code that uses that I guess to call the composable instead?